### ISSUE TITLE
Sometimes username sometimes full name used as email sender

### ISSUE THREAD

**Issue by oszem** [kanboard/kanboard#4881]
### Checklist

- [OK] I verified that Kanboard is correctly installed
- [OK] I checked that the problem does not come from a plugin
- [OK] I checked that the problem is not already reported

### Actual behaviour

Sometimes username sometimes full name in used as email sender for different users.  

### Expected behaviour

How or where can it be set that always one or the other is used for all users?

### Steps to reproduce

Take different users and receive notification via task creation or update.

### Screenshots

![image](https://user-images.githubusercontent.com/5314882/132455182-77fe5947-1660-403f-9d55-ce4c36c7f846.png)

### Configuration

- Kanboard version: 1.2.20
PHP version: 7.2.24-0ubuntu0.18.04.7
PHP SAPI: apache2handler
HTTP Client: cURL
OS version: Linux 4.15.0-137-generic
Database driver: postgres
Database version: 10.17 (Ubuntu 10.17-0ubuntu0.18.04.1)
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0

**Comment by oszem** [comment]
I tried to change in app/Helper: /*?: $user['username']*/;
and app/Notification/MailNotification: $user['name'] /*?: $user['username']*/,

no success

**Comment by fguillot** [comment]
Perhaps, you can tell us which email notifications doesn't have the correct sender?

**Comment by oszem** [comment]
Certainly! New comment, Assignee change, Task update, New attachment, Task closed, New position, mentioning, New subtask, Column change, ...
